docs/help-topics/rhtap-import-configure-component/rhtap-import-configure-component.yml

```diff
@@ -11,18 +11,23 @@
     - **Target port**: Is it correct? If not, click in the field to modify it.
     - **Dockerfile**: If you specified **Dockerfile** as your runtime, make sure we detected the right one. Click in the field if you need to modify it. Don't worry! We hide this field if you chose a different runtime.
-    - **Default build pipeline**: To specify how to trigger rebuilds, toggle to choose either the default build pipeline or a custom one.
-        - **Default build pipeline**: This runs faster because it makes only critical image checks. Consider starting here to make sure we can successfully build and deploy your component.
-        - **Custom build pipeline**: This is triggered when you make commits to your source code repository. This pipeline runs more checks and security scans than the default pipeline, but it takes a bit more time because it's more thorough. **NOTE:** To use a custom pipeline, you must be the owner of your repository so that you can authorize the installation of our application in your GitHub instance and then grant the app access to your repository. If someone else owns your repository, fork it, then go back to the Add components page and start again.
+    - **Default build pipeline**: This switch is on by default. Toggle the switch off to create a **Custom** build pipeline. **Default** and **Custom** build pipelines are triggered differently and include different tasks.
+        - **Default** build pipelines are triggered manually and run faster by skipping some security checks and tests.
+        - **Custom** build pipelines are triggered when you make commits to your source code repository. They run more tests than default pipelines, and you can customize which tasks are included. **NOTE:** To use a custom pipeline, you must be the owner of your repository so that you can authorize our application in your GitHub account and then grant it access to this repository. If someone else owns this repository, fork it, and use the new repo URL when adding the component.
     - **CPU** and **Memory**: Choose how many of each of these you want for your application, and in what unit, depending on your deployment requirements.
+    - **Instances**: How many instances of this image do you want to deploy into an environment? Each instance requires its own resources.
 
-    In the **Environment variables** section, enter a variable name and value to customize how we deploy your application.
+    In the **Environment variables** section, enter a variable name and value to customize how we deploy your application. 
+    
+    If you defined your runtime environment variables in a devfile, we automatically extract them. Check that we detected them correctly when you configure your components for deployment.
+
+    If you don’t use a devfile or it doesn’t include environment variables, you can add them manually in the **Environment variables** section fields.
     
     ***
     **WARNING**
```

Comment:
Do you know if https://issues.redhat.com/browse/HACDOCS-354 will add information about pipeline secrets to this procedure? Users can specify build time secrets during the import flow now. Opaque support is already implemented, and image pull secrets will be merged soon.

Reply:
Thanks for bringing this up. Afaik that issue will only expand an upstream guide, I've reached out to the writer to double-check and will add this information here.